src/types.ts

```diff
@@ -14,6 +14,8 @@ import penumbra from './API';
 import { PenumbraError } from './error';
 import { PenumbraZipWriter } from './zip';
 
+export { PenumbraZipWriter } from './zip';
```

Comment:
```suggestion
```

duplicated to line 15?

Reply:
I want to export this from the index. I'm not sure if this is the best syntax though

Reply:
ohhhhh sorry its import vs export, my b